docs/dev-guide/integration/rest-api/content/upload.rst

```diff
@@ -63,6 +63,13 @@ and provide offset information for Pulp to use when assembling it.
 
 | :return:`None`
 
+**Note:**
+* For full size of the file, offset should be set to 0.
```

Comment:
s/For full size of the file, offset should be set to 0./When uploading a whole file in a single request, you must still specify the offset of 0./